gum/backend-x86/gumstalker-x86.c

```diff
@@ -905,19 +968,274 @@ gum_stalker_init (GumStalker * self)
   }
 # endif
 #endif
+
+#ifdef HAVE_LINUX
+  gum_gxx_personality_v0_impl = gum_exec_ctx_find_gxx_personality_v0 ();
+  gum_unwind_find_fde_impl = gum_exec_ctx_find_unwind_find_fde ();
+#endif
+}
+
+#ifdef HAVE_LINUX
+
+static gum_gxx_personality_v0
+gum_exec_ctx_find_gxx_personality_v0 (void)
```

Comment:
The name `find_*` makes it sounds rather innocent, like it doesn't go ahead and also hook the function. My gut feeling is that we should merge these three functions into one named something like `gum_stalker_ensure_unwind_apis_instrumented`.